src/hooks/useBestTrade.ts

```diff
@@ -31,7 +31,7 @@ export function useBestTrade(
 
   const [debouncedAmount, debouncedOtherCurrency] = useDebounce(
     useMemo(() => [amountSpecified, otherCurrency], [amountSpecified, otherCurrency]),
-    200
+    600
```

Comment:
can you move this to a constant

Reply:
at the top of the file or in a constants file?